src/tabs/tabs.js

```diff
@@ -22,7 +24,7 @@ angular.module('mm.foundation.tabs', [])
 
   ctrl.addTab = function addTab(tab) {
     tabs.push(tab);
-    if (tabs.length === 1 || tab.active) {
+    if ($scope.openFirstTab && (tabs.length === 1 || tab.active)) {
```

Comment:
This logic isn't quite right. `openFirstTab` doesn't always open the first tab, it could open any tab that has `active` set to `true`. Perhaps we should change the option name to `openOnLoad` or something more generic like that.